From e7fb861749bfa8e6b26eb059c5eed157f2cdf2c0 Mon Sep 17 00:00:00 2001
From: Timo Jyrinki <timo.jyrinki@canonical.com>
Date: Mon, 4 Feb 2013 11:01:00 +0200
Subject: [PATCH] Add qdoc and qdoc3

Make qtchooser handle also qdoc and qdoc3, expected to be found
similar to other tools by developers.

Change-Id: Ie6bfb1d85fbdcb1919f107e97557ccc848f334cd
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
---
 Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 1b93597..6be98a6 100644
--- a/Makefile
+++ b/Makefile
@@ -13,6 +13,8 @@ TOOLS = assistant \
 	qdbuscpp2xml \
 	qdbusviewer \
 	qdbusxml2cpp \
+	qdoc \
+	qdoc3 \
 	qglinfo \
 	qhelpconverter \
 	qhelpgenerator \
-- 
1.7.1

